loris/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:14.04
 
-RUN apt-get update
+RUN apt-get update --fix-missing
```

Comment:
Mmm. Why do we need this? It should run afresh every time.

Is the issue that this layer is already cached locally, I wonder?

Reply:
It rebuilt for me and didn't work without this :(

Reply:
I think the correct fix is to tie this to the second line:

```
RUN apt-get update &&
    apt-get install --yes ...
```

so we get an update *and* an install in the same, cached step.

(Let’s talk about exactly why this works when you’re free.)

Reply:
I think I can guess, just being lazy!

Reply:
:D